config/routes.rb

```diff
@@ -10,6 +10,7 @@
   post '/login', to: "session#create"
   delete '/logout', to: "session#destroy"
   get '/dashboard', to: "users#show"
+  post '/checkout', to: "orders#new"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.